app/controllers/admin/admin_users_controller.rb

```diff
@@ -144,11 +147,43 @@ def update_user
 
         @user.save
         flash[:notice] = success_message
-        redirect_to request.referer || root_path
+        if params[:admin_action] == 'spamban'
+          redirect_to confirm_delete_user_creations_admin_user_path(@user)
+        else
+          redirect_to request.referer || root_path
+        end
 
       end
     end
   end
+  
+  before_filter :user_is_banned, :only => [:confirm_delete_user_creations, :destroy_user_creations]  
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Trailing whitespace detected.